db/migrate/20201110140112_add_provider_id_to_trainees.rb

```diff
@@ -0,0 +1,7 @@
+class AddProviderIdToTrainees < ActiveRecord::Migration[6.0]
+  # rubocop:disable Rails/NotNullColumn
```

Comment:
Can we do this in `.rubocop.yml` and make it global?

Reply:
Forgot to do that - good shout.